src/content/decentralized-identity/index.md

```diff
@@ -118,6 +122,10 @@ Here's an example to show how on-chain attestations might work in practice:
 
 4. Decentralized identity makes identity data portable. Users store attestations and identifiers in mobile wallet and can share with any party of their choice. Decentralized identifiers and attestations are not locked into the database of the issuing organization.
 
+5. Decentralized identity should work well with emerging zero-knowledge technologies that will enable individuals to prove they own or have done something without revealing what that thing is. This could become a powerful way to combine trust and privacy for applications such as voting.
+
+6. Decentralized identity enables anti-Sybil mechanisms that can identify when one individual human is pretending to be multiple humans in order to game or spam some system.
```

Comment:
```suggestion
5. Decentralized identity should work well with emerging zero-knowledge technologies that will enable individuals to prove they own or have done something without revealing what that thing is. This could become a powerful way to combine trust and privacy for applications such as voting.

6. Decentralized identity enables anti-Sybil mechanisms to identify when one individual human is pretending to be multiple humans to game or spam some system.
```